Runtime/UI/SafeArea.cs

```diff
@@ -0,0 +1,95 @@
+using UnityEngine;
+
+namespace Graphy.Runtime.UI
+{
+    /// <summary>
+    ///     Component that matches the size of the RectTransform to the safe area.
+    /// </summary>
+    [ExecuteAlways]
```

Comment:
Is there a reason we need to execute this while in edit mode?

Reply:
Mobile app developers use Device Simulator to check safe area in Editor. Therefore, I want to apply this script in Edit Mode as well.

![image](https://user-images.githubusercontent.com/47441314/201000589-a64e7d53-0c1b-4877-ba22-6383574ed053.png)